Cargo.Bazel.toml.lock

```diff
@@ -10002,7 +10014,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "29ad2e15f37ec9a6cc544097b78a1ec90001e9f71b81338ca39f430adaca99af"
 dependencies = [
  "libc",
- "mio",
+ "mio 0.8.10",
```

Comment:
looks like we might have a dependency on two different `mio` versions. Any way to work around that?

Reply:
yes, updating actix-server. This is transitive dependency, so the only thing we can do is keep everything to the most recent version.

I will send a follow-up MR.